packages/sdk/src/swap/v2/types.ts

```diff
@@ -114,10 +112,7 @@ interface ReceivingVaultSwap extends VaultSwapCommonFields {
 }
 
 interface SwappingVaultSwap extends ReceivingVaultSwap {
-  swap:
-    | Exclude<SwapChunk, 'latestSwapRescheduledAt' | 'latestSwapRescheduledBlockIndex'>
-    | undefined;
-  swapEgress: EgressFields | undefined;
+  swap: SwapChunk | undefined;
```

Comment:
contract swaps can be rescheduled if the global price impact protection hits

Reply:
not sure if we gain anything from separate contract swaps from channel swaps in the types now, but can revisit that later